server/app/boot_jobs.rb

```diff
@@ -5,6 +5,7 @@
 unless ENV['RACK_ENV'] == 'test'
   MongoPubsub.start!(PubsubChannel.collection)
   JobSupervisor.run!
+  Mongodb::Migrator.new.migrate
```

Comment:
Puma hooks may not be a good spot if you want to do something like `irb -r ./server.rb`